src/SFA.DAS.EmployerDemand.MockServer/restart-demand.json

```diff
@@ -14,10 +14,10 @@
         -0.773876
       ]}
   },
-  "restartDemandExists" : false,
-  "emailVerified" : false,
+  "restartDemandExists" : true,
+  "emailVerified" : true,
   "organisationName" : "test org",
-  "contactEmail" : "test@test.com",
```

Comment:
Might be worth us creating a tech task to be able to set our own keys and have a dev decryptor, otherwise it will be hard to create acceptance tests for all these scenariios